### DISCUSSION TITLE
Help needed for translation

### DISCUSSION THREAD

**Discussion by ldmpub** [freeplane/freeplane#229] [General]
Hi,

Can I have more details on the following string because yet I can not translate it (to french).
I have only understood that its used in the context of "filters" but I'm unable to locate this string in any Freeplane screen ...

https://hosted.weblate.org/translate/freeplane/translation/fr/?checksum=5a86e28168baa30f 

Thanks for your help,

Note: it's the only string not translated yet to french which prevents to reach 100% achievement for french translation

**Comment by ldmpub** [comment]
Ping ... 😋 does this 'new' discussion forum is monitored by Freeplane dev team ?

**Reply by ldmpub** [reply]
Or should I post this question / request somewhere else?

Thanks in advance for your feedback 👍

**Reply by dpolivaev** [reply]
The string is used in a new element of filter composer, see screenshot
![Selection_038](https://user-images.githubusercontent.com/1558052/132955546-c330ce42-99ff-4cf6-b307-da5305bba41c.png)

**Comment by xarich** [comment]
Thanks, translation done with something like: "Node itself" 😎.
